cyhy/mailer/cli.py

```diff
@@ -203,6 +225,7 @@ def get_requests_raw(db, query, batch_size=None):
         "agency.contacts.name": True,
         "agency.contacts.email": True,
```

Comment:
Alphabetization since we're here.

```suggestion
        "agency.contacts.email": True,
        "agency.contacts.name": True,
```